src/Snapshotting/Tests/LightSwitchTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace EventSauce\EventSourcing\Snapshotting\Tests;
+
+use EventSauce\EventSourcing\AggregateRootId;
+use EventSauce\EventSourcing\AggregateRootTestCase;
+
+class LightSwitchTest extends AggregateRootTestCase
+{
+    protected function newAggregateRootId(): AggregateRootId
```

Comment:
Nitpick, in the other test, helper functions like these come last.

Reply:
Says who? 😂